### PR TITLE
[FEATURE][86dqv7bbn] Cancer Caregiver Survey

### PR DESCRIPTION
**Pull request by cgarcia-lightit**
# Light It Pull Request Template ⚡⚡⚡

_Resolve_: [86dqv7bbn](https://app.clickup.com/t/86dqv7bbn)

## Description and Requires ⭐

feature: add caregiver survey for profiled caregivers

- [ ] This change requires a database update or migration
- [ ] This change requires a documentation update

## Checklist ✅

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/eo_web/src/App.tsx [hunk 1]
```diff
@@ -24,7 +24,8 @@ interface EnvironmentsConfigs {
   CANCER_INTRO_QUESTION_CAREGIVER_ID: string;
   CANCER_PROFILE_PATIENT_ID: string;
   CANCER_PROFILE_CAREGIVER_ID: string;
-  CANCER_SURVEY_FORM: string;
+  CANCER_PATIENT_SURVEY_FORM: string;
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/App.tsx:27]: Shouldn't this be named `CANCER_PATIENT_SURVEY_ID` ?

File: apps/eo_web/src/screens/Cancer/SurveyForm.tsx [hunk 2]
```diff
@@ -1,33 +1,44 @@
...
 
-
-
-
 export const SurveyForm = () => {
   const [searchParams] = useSearchParams();
-
   const email = searchParams.get("email") || "";
+  const profiled = searchParams.get("profiled") || "patient";
+  const symptoms = searchParams.get("symptoms") || "";
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/screens/Cancer/SurveyForm.tsx:16]: I think it's better to use the `??` operator instead of `||`.

**Review by sgoycoechea-lightit — APPROVED** [review]

File: apps/eo_web/src/screens/Cancer/SurveyForm.tsx (hunk 2 above)
Comment (cgarcia-lightit) [reply apps/eo_web/src/screens/Cancer/SurveyForm.tsx:16]: Done.

File: apps/eo_web/src/App.tsx (hunk 1 above)
Comment (cgarcia-lightit) [reply apps/eo_web/src/App.tsx:27]: solve.

**Pull request merged**